pkg/kube/snapshot/snapshot_alpha.go

```diff
@@ -353,14 +355,16 @@ func (sna *SnapshotAlpha) getDeletionPolicyFromClass(snapClassName string) (stri
 	return vsc.DeletionPolicy, nil
 }
 
-func UnstructuredVolumeSnapshotAlpha(name, namespace, pvcName, contentName, snapClassName string, labels map[string]string) *unstructured.Unstructured {
+// UnstructuredVolumeSnapshotAlpha has contentMeta has metadata of VolumeSnapshotContent.
+// If contentMeta has name value set, UnstructuredVolumeSnapshotAlpha will create both VolumeSnapshot and VolumeSnapshotContent resource.
```

Comment:
```suggestion
// UnstructuredVolumeSnapshotAlpha creates Unstructured object for the VolumeSnapshot resource.
```
Comment needs correction, `UnstructuredVolumeSnapshotAlpha` doesn't create any object, it just returns unstructured  object for the given resource

Reply:
Same for beta implementation of the function